corehq/apps/cloudcare/static/cloudcare/js/util.js

```diff
@@ -121,6 +130,28 @@ hqDefine('cloudcare/js/util', function () {
         NProgress.done();
     };
 
+    var reportFormplayerErrorToHQ = function (data) {
+        try {
+            var reverse = hqImport("hqwebapp/js/initial_page_data").reverse;
+
+            $.ajax({
+                type: 'POST',
+                url: reverse('report_formplayer_error'),
+                data: JSON.stringify(data),
+                contentType: "application/json",
+                dataType: "json",
+                success: function () {
+                    console.info('Successfully reported error: ' + JSON.stringify(data));
+                },
+                error: function () {
+                    console.error('Failed to report error: ' + JSON.stringify(data));
+                },
+            });
+        } catch (e) {
+            console.error("reportFormplayerErrorToHQ failed hard and there is no where else to report this error", e);
```

Comment:
Maybe include `data` in this error message? It seems unlikely that the `error` callback will fire if there is an exception caught here.
```suggestion
            console.error(
                "reportFormplayerErrorToHQ failed hard and there is nowhere " +
                "else to report this error: " + JSON.stringify(data),
                e,
            );
```